endpoints/getting-started/client/main.go

```diff
@@ -26,150 +23,103 @@ import (
 )
 
 var (
-	host   = flag.String("host", "", "The API host. Required.")
-	apiKey = flag.String("api-key", "", "Your API key. Required.")
-
-	echo           = flag.String("echo", "", "Message to echo. Cannot be used with -service-account")
-	serviceAccount = flag.String("service-account", "", "Path to service account JSON file. Cannot be used with -echo.")
+	host                = flag.String("host", "", "The API host. Required.")
+	audience            = flag.String("audience", "", "The audience for the JWT. equired")
+	serviceAccountFile  = flag.String("service-account-file", "", "Path to service account JSON file. Required.")
+	serviceAccountEmail = flag.String("service-account-email", "", "Path email associated with the service account. Required.")
 )
 
 func main() {
 	flag.Parse()
 
-	if *apiKey == "" || *host == "" {
-		flag.PrintDefaults()
-		os.Exit(1)
-	}
-	if *serviceAccount == "" && *echo == "" {
-		fmt.Fprint(os.Stderr, "Provide one of -echo or -service-account.")
-		flag.PrintDefaults()
-		os.Exit(1)
-	}
-	if *serviceAccount != "" && *echo != "" {
-		fmt.Fprint(os.Stderr, "Provide only one of -echo or -service-account.")
-		flag.PrintDefaults()
+	if *audience == "" || *host == "" || *serviceAccountFile == "" || *serviceAccountEmail == "" {
+		fmt.Println("requires: --host, --audience, --service-account-file, --service-account-email")
 		os.Exit(1)
 	}
-
-	var resp *http.Response
-	var err error
-	if *echo != "" {
-		resp, err = doEcho()
-	} else if *serviceAccount != "" {
-		resp, err = doJWT()
-	}
+	jwt, err := generateJWT(*serviceAccountFile, *serviceAccountEmail, *audience, 3600)
 	if err != nil {
 		log.Fatal(err)
 	}
-	b, err := httputil.DumpResponse(resp, true)
+	fmt.Println(jwt)
```

Comment:
Should this have any extra context? Something like `fmt.Printf("JWT: %+v", jwt)`? Maybe `"%v"` if you don't want field names.

Reply:
I decided to just remove it. It's encrypted data, so not particularly useful beyond debugging anyway